src/tasks/installers/operator.ts

```diff
@@ -149,6 +149,21 @@ export class OperatorTasks {
           }
         }
       },
+      {
+        title: 'Prepare Eclipse Che cluster CR',
+        task: async (ctx: any, task: any) => {
+          const cheCluster = await kube.getCheCluster(flags.chenamespace)
+          if (cheCluster) {
+            ctx.cheClusterCreated = true
+            task.title = `${task.title}...It already exists..`
+          } else {
+            const yamlFilePath = flags['che-operator-cr-yaml'] === '' ? ctx.resourcesPath + 'crds/org_v1_che_cr.yaml' : flags['che-operator-cr-yaml']
+            ctx.CR = yaml.safeLoad(fs.readFileSync(yamlFilePath).toString())
```

Comment:
basically ctx.CR and ctx.cheClusterCreated are the same, right?

Reply:
Yes, I removed unused `ctx.cheClusterCreated `